contracts/src/libraries/constants/T1Constants.sol

```diff
@@ -16,6 +16,9 @@ library T1Constants {
     /// @notice Chain ID of the T1 devnet
     uint64 internal constant T1_DEVNET_CHAIN_ID = 299_892;
 
+    /// @notice Chain ID of the PR1 network
+    uint64 internal constant PR1_CHAIN_ID = 84_532;
```

Comment:
This naming is ambiguous to me. I think we're better off being explicit about the network name in the variable

Reply:
Something like `BASE_SEPOLIA_CHAIN_ID`? I've followed what there was in this file. `L1_CHAIN_ID` is another candidate to be more specific as `SEPOLIA_CHAIN_ID`?

Reply:
the difference is there is only one L1 (Ethereum) and one T1, but many partner rollups.

For me those names are fine

Reply:
There is also mainnet L1 vs testnet L1. The following partner rollup would be PR2. I suggest to revisit the purpose of this file and the var names when we introduce more PRs or we approach mainnet.